Source/hydro/Castro_mol_hydro.cpp

```diff
@@ -90,17 +94,12 @@ Castro::construct_mol_hydro_source(Real time, Real dt, MultiFab& A_update)
         const Box& obx = amrex::grow(bx, 1);
         const Box& obx2 = amrex::grow(bx, 2);
 
-        FArrayBox &statein  = Sborder[mfi];
-        Array4<Real const> const uin_arr = statein.array();
-
-        FArrayBox &stateout = S_new[mfi];
+        Array4<Real const> const uin_arr = Sborder.array(mfi);
```

Comment:
Stylistically, I strongly prefer Sborder[mfi].array() to Sborder.array(mfi)

Reply:
there are > a dozen instances in that file already that do `.array(mfi)` already, so this is consistent with that.
I'd be fine with a follow-on PR that standardizes all of the code to the alternate approach.